### ISSUE TITLE
Response Json

### ISSUE THREAD

**Issue by marinofaggiana** [rullzer/ioshelper#1]
The first record should be the self directory, but I don't understand what's this :

<img width="523" alt="Schermata 2020-03-14 alle 10 22 57" src="https://user-images.githubusercontent.com/8616947/76679104-fe7cf000-65dd-11ea-931d-45f60cb4175a.png">

**Comment by rullzer** [comment]
I'm going to need a bit more information.

How does your file tree look like.
What is the full request you did

**Comment by marinofaggiana** [comment]
This is the first response item of the request for "/" : 

"name" = .jpg ?
"directory" = 0 ?
"mimetype" = application/octet-stream ?

**Comment by rullzer** [comment]
returning the folder itself feels weird. Because then it is no longer a flat list. But there is a hierarchy.
Also because the name will not match etc.

**Comment by marinofaggiana** [comment]
? please ?

**Comment by rullzer** [comment]
? please what?
I don't get the question.

We can return the current folder as well sure. BUt then we need to introduce a hierarchy as well to the response. Just adding it in there doesn't work IMO.

**Comment by marinofaggiana** [comment]
asked you what those strange data on the first record, I don't understand your answer

**Comment by rullzer** [comment]
well apparenbtly you have a file in your folder that is just a .jpg
This is sorted on top.
But this is not a valid file. Hence the mimetype.
directory = 0 (since it is a file)
and name = '.jpg' since that seems to be the name

**Comment by marinofaggiana** [comment]
very strange ... and is not normal, I don't know where it comes from but it doesn't exist, and it doesn't appear anywhere

**Comment by rullzer** [comment]
Create a new folder and see if you see it there as well.
